lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.0.0",
+  "lerna": "3.11.0",
   "version": "1.0.0-beta.17",
   "npmClient": "yarn",
   "useWorkspaces": true,
```

Comment:
You'll need to tell lerna to not publish to npm when running `lerna publish`. The actual publishing will happen in Shipit.

You'll need to add this:
https://github.com/Shopify/theme-scripts/blob/master/lerna.json#L9-L13

Reply:
for `yarn release`, I change it from `yarn publish` to `yarn version`

Reply:
Ah right because we're on `3.x` now.